vaccine-standby/assets/index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Get started with your Twilio Functions!</title>
+    <link rel="stylesheet" href="//fonts.googleapis.com/css?family=Roboto:300,300italic,700,700italic">
+    <link rel="stylesheet" href="https://unpkg.com/normalize.css/normalize.css">
+    <link rel="stylesheet" href="https://unpkg.com/milligram/dist/milligram.min.css">
+    <link rel="stylesheet" href="/styles.css">
+  </head>
+  <body>
+    <header>
+      <h1>COVID-19 Vaccine Standby List</h1>
+      <img class="standby-list-diagram" src="/standby-list-diagram.png" />
+      <p>This Quick Deploy app is designed for public health agencies to create a COVID-19 vaccine eligibility standby list for their city or state.
```

Comment:
Same here, you can remove the `Quick Deploy` since especially at this point it's irrelevant for people

Reply:
Done